command:
  type: velocity_and_yaw_const
  vel: 1.0
  yaw_rate: 0.1

motion_model:
  type: circle2d
  dt: 0.1
  std_dev: [0.1, 0.1, 0.01745329251994329, 1.0]

observation_model:
  type: gps_xy
  std_dev: [1.0, 1.0]

noise_model:
  type: sample

gaussian_filter:
# type: ekf
  type: ukf
  alpha: 0.001
  beta: 2.0
  kappa: 0.0
